api/app/Policies/AssessmentStepPolicy.php

```diff
@@ -22,17 +21,15 @@ class AssessmentStepPolicy
      */
     public function create(User $user, $request)
     {
-        $poolId = null;
         if (array_key_exists('pool_id', $request)) {
             $poolId = $request['pool_id'];
-        } else {
-            return Response::deny('Cannot find pool.');
-        }
+            $pool = Pool::with('team')->find($poolId);
 
-        $pool = Pool::with('team')->find($poolId);
+            return $pool->getStatusAttribute() === PoolStatus::DRAFT->name
+            && $user->isAbleTo('update-team-draftPool', $pool->team);
```

Comment:
I know our linter doesn't seem to care, but I want to ask for this line to be indented  😅

Reply:
I changed this as part of readding denial messages to the policy but then I changed this into an if statement and I'm not sure I like it anymore.